### ISSUE TITLE
Missing publisher information in MSI

### ISSUE THREAD

**Issue by Firewolf1337** [glpi-project/glpi-agent#791]
### Bug reporting acknowledgment

Yes, I read it

### Professional support

Yes, I know

### Describe the bug

The MSI Package of 1.11 seems to have no publisher information or rather it is not signed with a certificate.
![image](https://github.com/user-attachments/assets/abab7747-3a2d-4ece-930a-a5a27471f35f)

### To reproduce

Download the MSI of 1.11
Start with turned on MS Defender SmartScreen

### Expected behavior

Not showing this Popup

### Operating system

Windows

### GLPI Agent version

v1.11

### GLPI version

10.0.15

**Comment by g-bougard** [comment]
Hi @Firewolf1337 

this is not really a bug, but still a packaging issue.

We are of course aware of this limitation. And indeed, first glpi-agent MSI packages was signed and we invest a lot of time in that feature. But, and I still spoke about this problem in other issues or discussions, when the time came to renew our code-signing certificate, our CA was NOT able to provide what we need in time. We even changed our CA and they also failed to honor their job. These CA didn't even provide any good reason to explain why they failed. As the related process has some costs we decided to get our money back and to stop loosing our time.

As we were blocked against this code-signing need to publish new versions, we decided to release MSI packages without code-signing. So yes, when you install manually using Windows UI, you'll obtain such blue windows alerting the MSI package is not signed. You won't have this alert if you install automatically the package.

So the final question will remain to be will you trust the package we provide. And that's up to you.

Glpi-agent remains an open-source solution. We provide even all the tool to build the MSI itself. If you require a signed package, it's also up to you to sign yourself the package you build yourself. You just need a code-signing certificate.

Anyway we didn't plan to fix this issue soon, but we are still open to fix it in the future.
